src/infra/main/template.yaml

```diff
@@ -152,21 +158,81 @@ Resources:
       Roles:
         - !Ref AISSampleLambdaHandlerRole
 
- #### TxMA ####
+  AccountDeletionProcessorSNSTopic:
```

Comment:
can we add a condition on this so that the topic is created in dev and build envs only, IsInternal possibly

Reply:
Hey @TaherKapasi , @domantas-gds and I discussed this with Jess yesterday. My understanding is that we do not have the ARN of topics that we need yet, so for now we have plugged this placeholder topic in all environments, so that deployment wouldn't fail if and when we go past dev/build (as we were using empty strings). cc. @domantas-gds 
The idea is to go back to the condition once we have arns for the higher envs

Reply:
Thanks Chiara!